main.go

```diff
@@ -152,29 +153,33 @@ func main() {
 	// Mount middleware
 	service.Use(middleware.RequestID())
 	// Use our own log request to inject identity id and modify other properties
-	service.Use(log.LogRequest(configuration.IsPostgresDeveloperModeEnabled()))
 	service.Use(gzip.Middleware(9))
 	service.Use(jsonapi.ErrorHandler(service, true))
 	service.Use(middleware.Recover())
 
 	service.WithLogger(goalogrus.New(log.Logger()))
 
-	publicKey, err := token.ParsePublicKey(configuration.GetTokenPublicKey())
-	if err != nil {
-		log.Panic(nil, map[string]interface{}{
-			"err": err,
-		}, "failed to parse public token")
-	}
-
 	// Setup Account/Login/Security
 	identityRepository := account.NewIdentityRepository(db)
 	userRepository := account.NewUserRepository(db)
 
 	appDB := gormapplication.NewGormDB(db)
 
+	publicKey, err := token.ParsePublicKey(configuration.GetTokenPublicKey())
+	if err != nil {
+		log.Panic(nil, map[string]interface{}{
+			"err": err,
+		}, "failed to parse public token")
+	}
 	tokenManager := token.NewManager(publicKey)
-	app.UseJWTMiddleware(service, jwt.New(publicKey, nil, app.NewJWTSecurity()))
+	// Middleware that extracts the token if present, otherwise ignore the request
+	jwtMiddlewareTokenContext := witmiddleware.TokenContext(publicKey, nil, app.NewJWTSecurity())
```

Comment:
@aslakknutsen no, the next handler will be executed. However, I kind of like more to throw an error in this middleware if there is an invalid token. The logs and error handling makes more sense doing it in this way.